src/external/wb_presets.c

```diff
@@ -2350,6 +2350,74 @@ const wb_data wb_preset[] =
   { "Nikon", "Coolpix P340", Cloudy, 0,			{ 1.976563, 1, 1.457031, 0 } },
   { "Nikon", "Coolpix P340", Flash, 0,			{ 2.085938, 1, 1.554688, 0 } },
 
+  // Nikon CoolPix P1000 firmware 1.2
+  { "Nikon", "COOLPIX P1000", HTMercury           , 0, { 1.76953125, 1, 1.61328125, 0 } },
```

Comment:
HTMercury is unknown. You may want to add it on top of the file in the Nikon section to allow translation.